src/Keboola/StorageApi/Client.php

```diff
@@ -1468,6 +1467,7 @@ public function uploadSlicedFile(array $slices, FileUploadOptions $options, File
                     $slices[$key] = $gzFilePath;
                 }
             }
+            $newOptions->setFileName($newOptions->getFileName() . '.gz');
```

Comment:
Pokud už ten soubor bude mít `.gz` příponu tak ji to zdvojí. Ale to je asi ok.

Reply:
Nevím o tom, že by nějaká komponenta dodávala komprimovaný slajsy - s těma je ve většině případů stejně potřeba něco v procesorech dělat, a ty s komprimovanejma souborama nepracují. Řekl bych, že komponenta má vždycky dodat nekomprimovaný data - my s tím pak můžeme dělat další optimalizace - třeba nařezat na optimální slice.

Reply:
Zas na druhou stranu u neslicovanejch souborů se kontroluje, jestli už náhodou nejsou komprimovaný a pak se nekomprimují a `.gz` se nepřidá. 

U sliců je to trochu složitější - komprimují se slicy (u každýho z nich se kontroluje, jestli už není komprimovanej). Udělal bych asi i kontrolu na už existující příponu u `fileName` slicovanýho souboru a pak to celý otestoval se všema možnejma scénářema (to bude asi trochu oříšek?).

Reply:
nechal bych to tak jak je to teď. A jenom přidal ten test na přidání toho `.gz` ke jménu souboru.

Reply:
je to tam